.github/workflows/snyk.yml

```diff
@@ -0,0 +1,15 @@
+name: Snyk Monitor Scanning
+on:
+ release:
+   types: [published] 
+ push:
+    branches:
+      - 'main' 
+      - 'master'
+ workflow_dispatch:
+
+jobs:
+  snyk-scan-ci:
+    uses: 'grafana/security-github-actions/.github/workflows/snyk_monitor.yml@main'
+    secrets:
+      SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
Is this a organization secret or do we need to add something to the repository?

Reply:
@rfratto This is an Org secret. No action is needed.